deployment/vm/manager.go

```diff
@@ -154,3 +166,10 @@ func (m *manager) createAndRecordVM(agentID string, stemcell bistemcell.CloudSte
 
 	return cid, nil
 }
+
+func (m *manager) setVMNetworkInterfaces(vmNetworkInterfaces map[string]biproperty.Map, currentIP string, networkInterfaces map[string]biproperty.Map) {
+	for networkName, networkInterface := range networkInterfaces {
+		networkInterface["ip"] = currentIP
```

Comment:
that's going to override ip field for all networks including VIP networks i believe.